data/custom/custom_renewables/TEMPLATE_csp_2050_0.091_installable.csv

```diff
@@ -0,0 +1,36 @@
+Generator,p_nom_max,annualcostEuroPMW,fixedomEuroPKW,installedcapacity,lifetime
+TR.11_1_AC,1,280787.0,40.0,0.0,30.0
+TR.12_1_AC,1,280787.0,40.0,0.0,30.0
+TR.13_1_AC,1,280787.0,40.0,0.0,30.0
```

Comment:
I wonder if we can be more generic in the templates? Would be good to keep all of them as light as possible similar as it's done in `custom_powerplants.csv` or in `custom_res_installable.csv`. Do you think it's possible to shorten the templates files keeping only a line or a few and as few values as possible?